index.html

```diff
@@ -3,382 +3,408 @@
-
-				<div class="framework-logo-div">
-					<div class="framework-logo">
-						<img src="Assets/Icons/icon-framework.svg" alt="Language Icon" />
-					</div>
-				</div>
-
-				<h3 class="skill-card-heading">Frameworks</h3>
-				<div class="skill-card-details">
-					<div class="skill-card-languages">
-						<ul class="skill-language-list">
-							<li>Bootstrap</li>
-							<li>Ruby on rails</li>
-							<li>Rspec</li>
-							<li>Capybara</li>
-							<li>Selenium</li>
-						</ul>
-					</div>
-				</div>
-
-			</section>
-
-			<section class="skill-card">
-
-				<div class="framework-logo-div">
-					<div class="framework-logo">
-						<img src="Assets/Icons/icon-skills.svg" alt="Language Icon" />
-					</div>
-				</div>
-
-				<h3 class="skill-card-heading">Skills</h3>
-				<div class="skill-card-details">
-					<div class="skill-card-languages">
-						<ul class="skill-language-list">
-							<li>Codekit</li>
-							<li>Github</li>
-							<li>Codepen</li>
-							<li>Gitlab</li>
-							<li>Terminal</li>
-						</ul>
-					</div>
-				</div>
-
-			</section>
-
-
-
-		</div>
-
-	</section>
-
-	<section class="contact-form-page" id="contact">
-		
-		<div class="form-div">
-
-		<h3 class="form-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in
-			touch.</h3>
-		
-			<form action="https://formspree.io/f/mgeqkdqj" method="post">
-				<input 
-					type="text" 
-					class="form-field" 
-					name="name" 
-					id="name" 
-					placeholder="Full name" 
-					required maxlength="30" >
-				
-				<input
-					type="email" 
-					class="form-field"
-					name="email"
-					id="email" placeholder="Email Address" 
-					required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$">
-			
-				<textarea 
-					name="message" 
-					id="message" 
-					cols="30" 
-					rows="10" 
-					placeholder="Enter text here" 
-					required 
-					maxlength="500"></textarea>
-				
-				<input type="submit" class="form-button" value="Get in Touch"  >
-			</form>
-		</div>
-		
-		<div class="footer-social-media">
-			
-			<ul class="footer-social-media-list">
-				<li>
-					<a href=""><img src="Assets/Icons/twitter.svg" alt="Twitter" /></a>
-				</li>
-
-				<li>
-					<a href=""><img src="Assets/Icons/Linkedin-icon.svg" alt="Linked In" /></a>
-				</li>
-
-				<li>
-					<a href=""><img src="Assets/Icons/medium.svg" alt="Medium" /></a>
-				</li>
-
-				<li>
-					<a href=""><img src="Assets/Icons/github.svg" alt="Github" /></a>
-				</li>
-
-				<li>
-					<a href=""><img src="Assets/Icons/angellist.svg" alt="Angel List" /></a>
-				</li>
-
-			</ul>
-			
-			<a class="footer-email" href = "mailto: abc@example.com">moseswamae18@gmail.com</a>
-	
-		</div>
-	</section>
-
-</body>
-
-</html>
\ No newline at end of file
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Wamae-Software Engineer Portfolio Homepage</title>
```

Comment:
Kindly note that the title of the Page is too long, please make it brief 🤗

For example:
- Wamae's Portfolio
- Wamae Full Stack Developer